### ISSUE TITLE
When the startCommand terminates, let us restart via the UI

### ISSUE THREAD

**Issue by oodavid** [stackblitz/core#2836]
**Is your feature request related to a problem? Please describe:**
Sometimes our dev server carks it and we need to re-run `npm run dev` to get going again.

**Describe the solution you'd like:**
I know we can open the terminal and run the command again, but I'd prefer that the UI said "your command has terminated, restart?" - maybe this sits alongside the `Commit n changes` button?

**Describe alternatives you've considered:**
Open the terminal and go again

**Additional context:**
none

**Comment by samdenty** [comment]
Good idea, not sure how common though. We could show a notification if autostart process crashes, or we could have a button inside the autostart terminal
